docs/architecture.md

```diff
@@ -165,7 +165,7 @@ metadata:
     cluster.x-k8s.io/control-plane: "true"
     cluster.x-k8s.io/cluster-name: "test1"
 spec:
-  version: v1.18.0
+  version: v1.20.10
```

Comment:
This is the kubernetes version :sweat_smile:

Reply:
Oops. Sorry :rofl: